core/backend/src/Model.ts

```diff
@@ -242,11 +242,18 @@ export class GeometricModel extends Model {
   /** @internal */
   constructor(props: GeometricModelProps, iModel: IModelDb) { super(props, iModel); }
 
-  /** Query for the union of the extents of the elements contained by this model. */
+  /** Query for the union of the extents of the elements contained by this model.
+   * @deprecated use [[queryRange]].
```

Comment:
nit:

`nextversion.md` entry for this?

Reply:
yes, meant to do that - thanks.

Reply:
Done.